docs/api-overview.mdx

```diff
@@ -200,6 +325,41 @@ An expiration date is not the only control on whether a credential is valid. You
 Credential data is stored securely via a data privacy vault that encrypts and tokenizes the data. The data is only retrieved and decrypted for the brand that the <Tip type="user"/> agrees to share with.
 :::
 
+:::info Composite Credentials
+Composite credentials are credentials that are made up of other credentials. For example, an `FullNameCredential` is made up of `FirstNameCredential`, `MiddleNameCredential`(optional) and `LastNameCredential`. When issuing a composite credential, you must provide the child credentials as well.
```

Comment:
I very much think that just having a random rule that MiddleName is optional does not make sense. Are there any other random implicit rules like this for other composite creds? This subjective optional should be avoided. Explicit definitions are much more desirable.

Reply:
Line2 it's another example of optional data

Reply:
Hmm =/. How is this documented? How is one supposed to know this? I don't think it is in our json schemas?

Reply:
<img width="615" alt="image" src="https://github.com/UnumID/UnumID.github.io/assets/11665444/7812deab-0172-4f19-9d26-fd5490973130">


In our JSON schema we have required field when is required

Reply:
I agree that we should be explicit about which child credentials are required/not required when issuing a composite credential. I'm sure there are a lot more cases like Address, where obviously line2 can't be required because many addresses don't have it, or FullName, where many issuers probably don't collect middle name. Especially this early on we are going to need to be flexible in allowing issuers to issue what they have and not try to force unnecessary requirements on them.

Reply:
Thanks for providing the example Leo. As long as this is explicitly defined in the json schema I suppose I'm ok with it.